Formula/polymake.rb

```diff
@@ -0,0 +1,56 @@
+class Polymake < Formula
+  desc "Tool for computations in algorithmic discrete geometry"
+  homepage "https://polymake.org"
+  url "https://polymake.org/lib/exe/fetch.php/download/polymake-3.2r3.tar.bz2"
+  sha256 "8423dac8938dcd96e15b1195432f6a9844e8c34727c829395755a5067ce43440"
+
+  depends_on "ant"
+  depends_on "boost"
+  depends_on "gmp"
+  depends_on "mpfr"
+  depends_on "ninja"
+  depends_on "ppl"
+  depends_on "readline"
+  depends_on "singular"
+
+  resource "Term::Readline::Gnu" do
+    url "http://search.cpan.org/CPAN/authors/id/H/HA/HAYASHI/Term-ReadLine-Gnu-1.35.tar.gz"
```

Comment:
Use a `https://cpan.metacpan.org/` URL here rather than this one.

Reply:
I'm not really familiar with perl stuff, is this something we might want to audit similar to `files.pythonhosted`?

Reply:
@vszakats mostly stamped out the existing examples in https://github.com/Homebrew/homebrew-core/pull/28144 and a follow-up PR, so hopefully people will stop finding/using that style of URL naturally, but if not, it's potentially a job for `audit`, aye.

Reply:
replaced with a link to cpan.metacpan.org